### DISCUSSION TITLE
What is the future Roadmap of Laragon?

### DISCUSSION THREAD

**Discussion by Rollo62** [leokhoa/laragon#597] [General]
Hi there,

I stumbled over Laragon last year and was immediately fascinated after a few tests.
Then I was going to try to support this project and looked a bit deeper into it.
Unfortunately, I found the following statement:
[457508](https://medium.com/@leovn/how-i-created-laragon-and-the-current-status-of-laragons-development-bc927c5ce1d6)
Which completely put me on hold.

I was checking the status of Laragon from time to time, but it seems to stuck on Version 6.0.
Meanwhile, many other new developments of Laragon 10, Valet and also Valet for Windows appeared.

Now again, the situation seems the same.
I still believe in Laragon and would like to use it for new projects, but are blocked by the current, unclear Roadmap.

Is there any plan to keep this great project alive, maybe be external participants, or will it slowly die out?

Please let me know, if there were any future plans, or how other developers deal with this situation.
Maybe there were already more active forks available, but I would like to stick with the original version and also like to 
donate for that, but unforunately I cannot with such unclear future perspective.

**Comment by mikailhamzat** [comment]
Following this psot

**Comment by alxndrv-alex** [comment]
I'll stand here awaiting any comments from @leokhoa

**Reply by RobiNN1** [reply]
I noticed that he ignores such posts because laragon doesn't require constant updates. You can update everything yourself (php, apache, nodejs...).

IMO it's dumb to think that it's dead just because there are not frequent updates.

**Comment by leokhoa** [comment]
Laragon is designed for longevity; it does not require frequent updates.

**Reply by alxndrv-alex** [reply]
Thanks for replying Leo! Yeah, that is one of many aspects why we love it :)

**Comment by AWqxKAWERbXo** [comment]
Hey @Rollo62 @alxndrv-alex @mikailhamzat ,

you might already noticed that things changed with Laragon 7. With Laragon 7, it will now be a tool for professionals with an upcoming option for semi-professionals or open source developers.

At the moment, the [GitHub Issues](https://github.com/leokhoa/laragon/issues) shows which Features or Bugs will be addressed first for the near future.

I'm sure that more plans will be revealed later this year.

**Reply by Rollo62b** [reply]
Sure, I have seen this.
Unfortunately I have little time to test and the big advantage of the portable design is in the new version somewhat gone.
From my first look into this, its still very useful to me, I will look into this more deeply, trying some projects and tests, and will very likely join this project soon.
I would have preferred to make donations, instead of now a more business oriented approach, which might stop possible new clients in the first place.
Laravel is moving very fast, also the whole ecosystem, while Laragon is still somewhat slower to move and I'm not seeing much acceleration.
I know how to deal with Laragon 6, but if Laragon 7 builds extra hurdles, that wouldn't push me towards it, only after careful estimation.
